panel/src/components/Forms/Blocks/BlockSelector.vue

```diff
@@ -122,6 +122,9 @@ export default {
 		paste(e) {
 			this.$emit("paste", e);
 			this.close();
+		},
+		close() {
+			this.$emit("cancel");
```

Comment:
`Nitpick:` emitting `close` could be semantically more correct, functionally I think both work

Reply:
No yeah, I think you're right. Wasn't sure if this was going to work... changing.